students/mjchang/session04/trigrams.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+import random
+import string
+
+def read_book(filename):
+    text = []
+    #string.punctuation removes punctuation, found this on stackoverflow, still trying to understand it fully
+    translator = str.maketrans('','', string.punctuation) 
+    with open(filename) as book:
+        for row in book:   
+            row = row.strip() #removes carriage lines
+            if row.startswith("End of the Project Gutenberg EBook"):
+                break    
+    #second part of removing punctuation, combine with extend
+            text.extend(row.translate(translator).split(' ')) #use extend to add multiple words to list
+    return text            
+       
+      
+    for word in text: #capitalize the word "I"
```

Comment:
Because of the return statement that precedes it, this _for_ loop becomes unreachable.